lib/cli.rb

```diff
@@ -43,6 +43,9 @@ class Cli
   post do |global_options,command,options,args|
     if command.is_a?(GLI::Commands::Help) && !global_options[:version] && ARGV == ["help"]
 
+      Machinery::Ui.puts "\nFor more detailed information, open the man page or visit our " \
+        "wiki page: https://github.com/SUSE/machinery/wiki"
```

Comment:
It should say how the man page is opened and it should say that the primary documentation is there and that secondary information is on the Wiki (while "primary" and "secondary" are not the right terms to describe it to users).